internal/js/modules/k6/browser/common/element_handle.go

```diff
@@ -170,14 +169,51 @@ func (h *ElementHandle) clickablePoint() (*Position, error) {
 		return nil, fmt.Errorf("node is either not visible or not an HTMLElement: %w", err)
 	}
 
-	// Filter out quads that have too small area to click into.
-	var layoutViewport *cdppage.LayoutViewport
-	getLayoutMetrics := cdppage.GetLayoutMetrics()
-	if _, _, _, layoutViewport, _, _, err = getLayoutMetrics.Do(cdp.WithExecutor(h.ctx, h.session)); err != nil {
-		return nil, fmt.Errorf("getting page layout metrics %T: %w", getLayoutMetrics, err)
+	width, height, err := h.evaluateInnerWidthHeight()
+	if err != nil {
+		return nil, err
+	}
+
+	return filterQuads(width, height, quads)
+}
```

Comment:
It's worth enriching these errors with the `clickablePoint: ...` prefix (or something else you may see fit) to see a better trace when we get an error. Currently, they just relay what they get from `evaluateInnerWidthHeight` and `filterQuads`.

Reply:
@inancgumus do you think we can merge it as-is, or better to wait for @ankur22 to apply your suggestion? 🤔

Reply:
This (#4123) is an old issue, I'd wait for him for one more day.

Reply:
I've updated it in 93af43c44cfa1bf66462ab77b97b7d09e54355a6.